modis_tools/auth.py

```diff
@@ -54,9 +54,17 @@ def has_download_cookies(session):
         assert logged_in.domain == URLs.EARTHDATA.value
         assert datetime.fromtimestamp(logged_in.expires) > datetime.now()
         assert logged_in.value == "yes"
-
-        # data = cookies["DATA"]
-        # assert data.domain == URLs.RESOURCE.value
+        if "DATA" in cookies:
+            # specific to the cookies for LP DAAC source
+            data = cookies["DATA"]
+            assert data.domain == URLs.RESOURCE.value
+        elif "CIsForCookie_OPS" in cookies:
+            data = cookies["CIsForCookie_OPS"]
+            # specific to the cookies for NSIDC DAAC source
+            assert data.domain == URLs.NSIDC_RESOURCE.value
+        else:
+            # print(cookies)
+            raise KeyError("resource domain not found in cookies")
```

Comment:
Maybe this is a good place to add a prompt for the user like:
```suggestion
            raise KeyError("Data source not recognized. Please open an issue informing us of the desired data source.")
```